packages/babel-plugin-transform-runtime/src/helpers.js

```diff
@@ -32,6 +32,57 @@ export function hasMinVersion(minVersion, runtimeVersion) {
   );
 }
 
+export function verifyRuntimeVersion(
+  options,
+  moduleName,
+  runtimeVersion,
+  actualRuntimeVersion,
+) {
+  if (!actualRuntimeVersion) return;
+
+  const minActualVersion = semver.minVersion(actualRuntimeVersion).version;
+  const minVersion = semver.minVersion(runtimeVersion).version;
+
+  if (minActualVersion === minVersion) return;
+
+  const fixedOptions = JSON.stringify(
+    {
+      plugins: [
+        [
+          "@babel/plugin-transform-runtime",
+          { ...options, version: minActualVersion },
+        ],
+      ],
+    },
+    null,
+    2,
+  ).replace(/^/gm, "  ".repeat(2));
+
+  if (semver.gt(minActualVersion, minVersion)) {
+    console.warn(
```

Comment:
I'd be very hesitant to add new logging to a plugin in a non-major if it isn't opt-in. Some tools will treat any stderr output as an indication that something in the build has failed.